website/docs/r/alert_channel_datadog.html.markdown

```diff
@@ -0,0 +1,45 @@
+---
+subcategory: "Alert Channels"
+layout: "lacework"
+page_title: "Lacework: lacework_alert_channel_datadog"
+description: |-
+  Create and manage Datadog Alert Channel integrations
+---
+
+# lacework\_alert\_channel\_datadog
+
+The Datadog alert channel provides a unified view of your metrics, logs, and performance data combined with your cloud security data.
+
+To find more information about the Datadog alert integration, see the [Lacework support documentation](https://support.lacework.com/hc/en-us/articles/360036989953-Datadog).
+
+## Example Usage
+
+```hcl
+resource "lacework_alert_channel_" "ops_critical" {
```

Comment:
```suggestion
resource "lacework_alert_channel_datadog" "ops_critical" {
```

Reply:
in the future, we need to find better examples other than `ops_critical`, just to have more variety 😅